### PR TITLE
[feature] presentation service link

### PR DESCRIPTION
**Pull request by caseyrollins**
### Purpose
- Allows the presentation service to be accessed by users directly from the OSF.
- Addresses [#1418](https://github.com/CenterForOpenScience/osf.io/issues/1418) and [#2115](https://github.com/CenterForOpenScience/osf.io/issues/2115#issuecomment-85587127)

### Changes
- Adds the link to the presentation service to the Explore tab in the navbar.

screen shot 2015-03-24 at 11 57 44 am

### CODE REVIEW

File: website/templates/nav.mako [hunk 1]
```diff
@@ -27,6 +27,7 @@
                     <a href="#" class="dropdown-toggle" data-toggle="dropdown">Explore <b class="caret"></b></a>
                     <ul class="dropdown-menu">
                         <li><a href="/search/?q=*&filter=registration">Registry</a></li>
+                        <li><a href="/presentations">Presentations</a></li>
```
Comment (sloria) [inline website/templates/nav.mako]: Add a trailing slash to prevent a redirect.

Comment (sloria) [inline website/templates/nav.mako]: Also, can you add one to the "Public Activity" link while you're at it?

Comment (caseyrollins) [inline website/templates/nav.mako]: Yep. Wil do.

**Comment by lbanner** [comment]
Nice work @caseyrollins !!

**Pull request merged**